### PR TITLE
Fix missing closing tag in admin orders.php

### PR DESCRIPTION
**Pull request by DivaVocals**

### CODE REVIEW

File: admin/orders.php [hunk 1]
```diff
@@ -550,12 +550,12 @@ function couponpopupWindow(url) {
...
           </tr>
 <?php if (zen_not_null($order->info['cc_cvv'])) { ?>
           <tr>
             <td class="main"><?php echo ENTRY_CREDIT_CARD_CVV; ?></td>
-            <td class="main"><?php echo $order->info['cc_cvv'] . (zen_not_null($order->info['cc_cvv']) && !strstr($order->info['cc_cvv'],TEXT_DELETE_CVV_REPLACEMENT) ? '&nbsp;&nbsp;<a href="' . zen_href_link(FILENAME_ORDERS, '&action=delete_cvv&oID=' . $oID, 'NONSSL') . '" class="noprint">' . TEXT_DELETE_CVV_FROM_DATABASE . '</a>' : ''); ?><td>
+            <td class="main"><?php echo $order->info['cc_cvv'] . (zen_not_null($order->info['cc_cvv']) && !strstr($order->info['cc_cvv'],TEXT_DELETE_CVV_REPLACEMENT) ? '&nbsp;&nbsp;<a href="' . zen_href_link(FILENAME_ORDERS, '&action=delete_cvv&oID=' . $oID, 'NONSSL') . '" class="noprint">' . TEXT_DELETE_CVV_FROM_DATABASE . '</a>' : ''); ?></td> /*Fix stock Zen Cart HTML (missing closing tag)*/
```
Comment (drbyte) [inline admin/orders.php]: The comment at the end of these lines is not necessary.

**Comment by drbyte** [comment]
I updated the title of the Pull Request to be more descriptive of what this PR is about.

**Comment by DivaVocals** [comment]
Sorry.. It was late.. and my descriptive skillz had escaped me.. :smile:

File: admin/orders.php (hunk 1 above)
Comment (DivaVocals) [inline admin/orders.php]: My bad.. that was my own note.. forgot to take it out..

**Comment by drbyte** [comment]
:+1:

**Comment by zcwilt** [comment]
:+1:

**Comment by ajeh** [comment]
:+1:

**Pull request merged**